static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -158,4 +164,74 @@
             bottom: 8px;
         }
     }
+
+    .immersive-main-media__media & {
+        // & = .youtube-media-atom
+        padding-bottom: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        height: 100%;
+        z-index: 0;
+        &__immersive-interface {
```

Comment:
.youtube-media-atom__immersive-interface is added in youtube.scala.html when the atom is in immersive and main media